Cargo.toml

```diff
@@ -85,7 +85,7 @@ path = "libduckdb-sys"
 version = "0.10.1"
 
 [package.metadata.docs.rs]
-features = []
+features = ['vtab', 'chrono']
```

Comment:
Did you mean to commit this?

Reply:
Yes, it's what adds these modules to docs.rs

Reply:
They're presently missing from here: https://docs.rs/duckdb/latest/duckdb/#modules

Reply:
Cool thanks